New package: WaveSpec v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: WaveSpec
- Repository: https://github.com/shagun751/WaveSpec.jl
- Created by: @shagun751
- Version: v0.1.0
- Commit: 01d14548f0242a76a78ea5042f7ff1a070c355b4
- Reviewed by: @shagun751
- Reference: https://github.com/shagun751/WaveSpec.jl/issues/8#issuecomment-2217855940
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
First release
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598493c991793ba65d8fdcb15ef95e147a6a039c09c4c2bbd67f71d2209762b565b9ff8f1c5486a9ec3b9b7d567c280026b4c8e09a450985b5748c50db29e151c641b12ca1a731bbab3a269472f35c4f189157368eba73d535ca93f27416cc0ad91fd86fd3df0a6aa4c220a3d71ae63a38da54b46475e3ab0ba860df601e8a1e699 -->

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README

JuliaRegistrator committed:
New package: WaveSpec v0.1.0

UUID: 908d4cf6-a6c0-449c-92e4-852c8f288332
Repo: https://github.com/shagun751/WaveSpec.jl.git
Tree: b561683776a759ed1cbf7c28939b438dccf2228a

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

shagun751 commented:
@goerz 
As requested, I have added a basic description of the package.
I hope WaveSpec can now be considered for merge.
I appreciate your effort in maintaining the Julia resgistery.

goerz commented:
[noblock] Thanks! You’re good to go. However, you have to edit your comment to contain `[nonlock]`, or it will prevent the automerge

shagun751 commented:
Done! Rookie errors 😅
Thank you.

[noblock]
